Sources/ArcGISToolkit/Components/Popups/AttachmentsPopupElementView.swift

```diff
@@ -80,12 +80,12 @@ struct AttachmentsPopupElementView: View {
             }
         }
         .task {
-            try? await popupElement.fetchAttachments()
-            let attachmentModels = popupElement.attachments.reversed().map { attachment in
-                AttachmentModel(attachment: attachment)
+            if let attachmentModels = try? await popupElement.attachments.reversed().map({ 
+                AttachmentModel(attachment: $0)
+            }) {
+                viewModel.attachmentModels.append(contentsOf: attachmentModels)
+                isLoadingAttachments = false
             }
```

Comment:
I like this a little better because it always sets `viewModel.attachmentModels` and always sets `isLoadingAttachments` to false.
```suggestion
            let attachmentModels = try? await popupElement.attachments.reversed().map { attachment in
                AttachmentModel(attachment: attachment)
            }
            viewModel.attachmentModels.append(contentsOf: attachmentModels ?? [])
            isLoadingAttachments = false
```